src/libs/NumberCovert.js

```diff
@@ -0,0 +1,90 @@
+import lodashGet from 'lodash/get';
+import loadashTrim from 'lodash/trim';
+import loadashTrimStart from 'lodash/trimStart';
+import loadashToString from 'lodash/toString';
+import loadashIncludes from 'lodash/includes';
+import loadashStartsWith from 'lodash/startsWith';
+import loadashPadStart from 'lodash/padStart';
+import Log from './Log';
+import Config from '../CONFIG';
+import translations from '../languages/translations';
+
+function showErrorAndReturn(locale) {
+    // When a locale is entered which is not available, on production log an alert to server
+
+    if (Config.IS_IN_PRODUCTION) {
+        Log.alert(`${locale} was not found.`, 0, {}, false);
+    }
+
+    return `${locale} was not found.`;
+}
+
+/**
+ * Returns a locally converted phone number without the country code
+ *
+ * @param {String} locale eg 'en', 'es-ES'
+ * @param {String|Array} number
+ * @returns {string}
+ */
+function toLocalPhone(locale, number) {
+    const numString = loadashTrim(loadashToString(number));
+
+    let withoutPlusNum;
```

Comment:
Okay. I have incorporated this change in new commit. Actually my preferred style is also ternary and nested ternaries but since nested ternaries are not allowed in your style guidelines so i avoided single ones as well.

I have rectified this in new commit.